resources/views/tools/bread/edit-add.blade.php

```diff
@@ -96,7 +100,7 @@
                                            name="display_name_singular"
                                            id="display_name_singular"
                                            placeholder="{{ __('voyager::bread.display_name_singular') }}"
-                                           value="{{ $dataType->display_name_singular ?? $display_name }}">
+                                           value="{{ $display_name }}">
```

Comment:
This var is defined in the controller and is the first value when adding a BREAD.
I think you should check if adding a BREAD and if not, use the translated attribute.
But I'd prefer to have this in the controller, the definitions are already there.